package/Dockerfile

```diff
@@ -14,12 +14,12 @@ RUN zypper -n install autoconf bison curl cmake doxygen make git gcc-c++ flex Me
     tar gzip dbus-1-devel lsb-release graphviz-devel libnsl-devel libcurl-devel libjson-c-devel libacl-devel && \
     rm -rf /var/cache/zypp/*
 
-RUN curl -L https://github.com/longhorn/nfs-ganesha/archive/refs/tags/v4_20230419.tar.gz| tar zx \
-	  && curl -L https://github.com/nfs-ganesha/ntirpc/archive/v4.2.tar.gz | tar zx \
-	  && mv nfs-ganesha-4_20230419 nfs-ganesha-4.2 \
-	  && rm -r nfs-ganesha-4.2/src/libntirpc \
-	  && mv ntirpc-4.2 nfs-ganesha-4.2/src/libntirpc
-WORKDIR /nfs-ganesha-4.2
+RUN curl -L https://github.com/james-munson/nfs-ganesha/archive/refs/tags/v5.7-trial.tar.gz | tar zx \
+	  && curl -L https://github.com/nfs-ganesha/ntirpc/archive/v5.0.tar.gz | tar zx \
+	  && mv nfs-ganesha-5.7-trial nfs-ganesha-5.7 \
```

Comment:
Why using trial version? We need to use an official release.

Reply:
Can you submit a PR to https://github.com/longhorn/nfs-ganesha? Thank you.

Reply:
As Derek mentioned, we need to use our downstream version instead like SPDK.

Reply:
This is a trial build only in my fork, but it is using the V5-stable release.  "Trial" is only for my testing.  It has run e2e in https://ci.longhorn.io/job/private/jog/longhorn-tests-regression/5666 and 5678.

I would love to make a PR, but pushing to what branch?